students/minseok/westagram/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    name            = models.CharField(max_length=50)
+    email           = models.EmailField(max_length=128)
+    password        = models.CharField(max_length=256)
+    phone_number    = models.IntegerField()
+    user_name       = models.CharField(max_length=50)
+    website         = models.CharField(max_length=300)
+    introduce       = models.CharField(max_length=100)
+    profile_photo   = models.ImageField()
```

Comment:
```suggestion
    name          = models.CharField(max_length=50)
    email         = models.EmailField(max_length=128)
    password      = models.CharField(max_length=256)
    phone_number  = models.CharField(max_length=100)
    user_name     = models.CharField(max_length=50)
    website       = models.CharField(max_length=1000)
    introduce     = models.CharField(max_length=100)
    profile_imgae = models.CharField(mx_length=1000)
    created_at = models.DateTimeField(auto_now_add=True)
    updated_at = models.DateTimeField(auto_now=True)
```
1. 전화번호는 일반적으로 문자열로 간주 됩니다. 숫자 이외에도 '-'나 '+' 같은 기호가 들어갈 수 있습니다.
2. url 링크 주소 같은 경우 생각보다 긴 경우가 많아서 1000정도의 길이를 주면 좋을 것 같습니다.
3. photot보다는 image라는 단어를 더 많이 사용하고, 이미지 같은 경우 이미지를 직접 저장하는 것이 아닌, 이미지가 저장된 링크 주소를 저장하는 것이 일반적입니다.
4. 데이터의 이력 관리를 위해서 created_at, updated_at column을 추가해주세요.

```python
created_at = models.DateTimeField(auto_now_add=True)
updated_at = models.DateTimeField(auto_now=True)
```